packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -87,6 +87,27 @@ describe("Dropdown", () => {
         expect(dropdown.instance().focusedIndex).toBe(2);
     });
 
+    it("doesn't close on touch interaction with option", () => {
```

Comment:
Good call putting this inside dropdown.test.js instead of the files I was suggesting earlier.